plugins/org.eclipse.n4js.ts.model/model/Types.xcore

```diff
@@ -161,6 +161,8 @@ class TModule extends SyntaxRelatedTElement, TAnnotableElement {
  */
 class ComposedMemberCache {
 	contains transient TMember[] cachedComposedMembers
+	// The compose type ref of the member being cached
+	contains transient TypeRef composedTypeRef
```

Comment:
why containment?
if we use containment here, it is important to use TypeUtils.copyIfContained() before setting the value of `composedTypeRef`.